climada/hazard/trop_cyclone.py

```diff
@@ -1082,39 +1179,180 @@ def compute_angular_windspeeds(si_track, d_centr, mask_centr_close, model, cyclo
+    rho_air_const : float or None, optional
+        The constant value for air density (in kg/m³) to assume in the formulas from Holland 1980.
+        By default, the constant value suggested in Holland 1980 is used. If set to None, the air
+        density is computed from pressure following equation (9) in Holland et al. 2010.
+        Default: 1.15
 
+    Returns
+    -------
+    ndarray of shape (npositions, ncentroids)
+        containing the magnitude of the angular windspeed per track position per centroid location
+    """
+    _vgrad(si_track, gradient_to_surface_winds)
+    _rho_air(si_track, rho_air_const)
+    _B_holland_1980(si_track)
+    result = _stat_holland_1980(si_track, d_centr, close_centr_msk, cyclostrophic=cyclostrophic)
+    result *= gradient_to_surface_winds
     return result
 
+def _compute_angular_windspeeds_h08(
+    si_track: xr.Dataset,
+    d_centr: np.ndarray,
+    close_centr_msk: np.ndarray,
+    cyclostrophic: bool = False,
+    gradient_to_surface_winds: float = DEF_GRADIENT_TO_SURFACE_WINDS,
+    rho_air_const: float = DEF_RHO_AIR,
+):
+    """Compute (absolute) angular wind speeds according to the Holland 2008 model
+
+    Parameters
+    ----------
+    si_track : xr.Dataset
+        Output of `tctrack_to_si`. Which data variables are used in the computation of the wind
+        profile depends on the selected model.
+    d_centr : np.ndarray of shape (npositions, ncentroids)
+        Distance (in m) between centroids and track positions.
+    close_centr_msk : np.ndarray of shape (npositions, ncentroids)
+        For each track position one row indicating which centroids are within reach.
+    cyclostrophic : bool, optional
+        If True, don't apply the influence of the Coriolis force (set the Coriolis terms to 0).
+        Default: False
+    gradient_to_surface_winds : float, optional
+        The gradient-to-surface wind reduction factor to use. The wind profile is computed on the
+        surface level, but the clipping interval of the B-value depends on this factor.
+        Default: 0.9
+    rho_air_const : float or None, optional
+        The constant value for air density (in kg/m³) to assume in the formula from Holland 1980.
+        By default, the constant value suggested in Holland 1980 is used. If set to None, the air
+        density is computed from pressure following equation (9) in Holland et al. 2010.
+        Default: 1.15
+
+    Returns
+    -------
+    ndarray of shape (npositions, ncentroids)
+        containing the magnitude of the angular windspeed per track position per centroid location
+    """
+    _rho_air(si_track, rho_air_const)
+    _bs_holland_2008(si_track, gradient_to_surface_winds=gradient_to_surface_winds)
+    return _stat_holland_1980(si_track, d_centr, close_centr_msk, cyclostrophic=cyclostrophic)
+
+def _compute_angular_windspeeds_h10(
+    si_track: xr.Dataset,
+    d_centr: np.ndarray,
+    close_centr_msk: np.ndarray,
+    cyclostrophic: bool = False,
```

Comment:
`cyclostrophic` is unused in this model. Adding it as a parameter is detected as issue by the linter, and rightly so. Ideas (not mutually exclusive):
1. As the model is cyclostrophic, calling `_compute_angular_windspeeds_h10(cyclostrophic=False)` should emit a warning, right?
2. We can remove the parameter, add a `**kwargs` argument, and check if `cyclostrophic` appears in there, then emit an appropriate warning.

@ThomasRoosli

Reply:
After personal discussion: We will tackle this in a possible follow-up PR